src/pages/integrations/log-management/applications/haproxy.mdx

```diff
@@ -15,7 +15,7 @@ Configure Filebeat to ship logs from HAProxy to Logstash and Elasticsearch.
   <InstallIntegration/>
   ### Install Filebeat
 
-  <InstallFilebeat />
```

Comment:
I think all os options could be supported, although I doubt any one does

Reply:
ignore this - looked a bit harder - looks like windows support is more docker/cygwin